### PR TITLE
Some tweak to helm

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: deploy/k8s/chart/templates/kafka/020-PersistentVolumeClaim.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{{- if .Values.deployPostgres }}
```
Comment (bobmcwhirter) [inline deploy/k8s/chart/templates/kafka/020-PersistentVolumeClaim.yaml:1]: Is the Kakfa PVC dependent upon deploying postgres?

Comment (ctron) [reply deploy/k8s/chart/templates/kafka/020-PersistentVolumeClaim.yaml:1]: no … good catch

**Review by bobmcwhirter — COMMENTED** [review]
Inline question

**Review by bobmcwhirter — APPROVED** [review]
Approved, but be aware I know sweet FA about kafka deployment.

**Review by desmax74 — APPROVED** [review]

**Pull request merged**